share/sftp-to-share/cleanup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+
+if [[ ${#} < 5 ]]; then
+  echo "USAGE: bash ${0} uni-name --profile <AWS IAM profile> --region <AWS REGION>"
+  exit 1
+fi
+
+STACK_NAME="sftp-to-share"
```

Comment:
The README suggests this is an input argument however it is static in this script. In my case I chose to use the default "sam-app" stack name so the cleanup script does not work.

Reply:
Good call - I updated the README to reflect that.